### ISSUE TITLE
Feature request create new folders, add files s3 bucket

### ISSUE THREAD

**Issue by octavd** [IBM/jupyterlab-s3-browser#4]
Hello,

Is there a way or are you planning on realeasing a new feature for this extension regarding creating folders, adding files on the bucket that is currently connected?

![request](https://user-images.githubusercontent.com/33294589/73655728-4af81600-4697-11ea-8e45-4e57f40f098f.PNG)

**Comment by reevejd** [comment]
Thanks for your interest! This feature isn't available right now, but I'm planning on adding it soon.

**Comment by octavd** [comment]
Hello reevejd,

First of all, thank you for the response.
Secondly, do you have an estimated time until release, or? :D

Thanks!

**Comment by reevejd** [comment]
I'm aiming to have it available by the end of the month.

**Comment by octavd** [comment]
Great. Looking forward to it.

**Comment by octavd** [comment]
Hello reevejd,

Any news about this request?

**Comment by reevejd** [comment]
I've been unexpectedly busy, but it's still on my radar. I hope to pick this issue up soon

**Comment by jpays** [comment]
Hi reevejd

Thank you for the good work. 
Do you still plan to implement this？
This would be unvaluable to us.

**Comment by reevejd** [comment]
Yep, I still plan to implement it. I'll take a look this weekend and see how much work it is / when I expect it to be available.

**Comment by octavd** [comment]
Hello @reevejd ,

Any news about this request? It would be amazing to have this features implemented.

Best regards,
Octavian

**Comment by PhE** [comment]
It's working (folder and file creation)
this issue should be closed
